src/hooks/use-authentication/index.ts

```diff
@@ -41,26 +41,25 @@ const useAuthentication = (
 		onUnauthenticated,
 	}) as { data: Session; status: SessionStatus }
 
+	// Deriving booleans about auth state
+	const isLoading = status === 'loading'
+	const isAuthenticated =
```

Comment:
I've considered putting this in `state`...but I think this a sufficient change for now given that we want to access session data via react query in subsequent work. 

Trying to weigh the pros and cons.